src/participant/participant.ts

```diff
@@ -318,7 +244,10 @@ export default class ParticipantController {
   async getDatabaseQuickPicks(): Promise<NamespaceQuickPicks[]> {
     const dataService = this._connectionController.getActiveDataService();
     if (!dataService) {
-      this._queryGenerationState = QUERY_GENERATION_STATE.ASK_TO_CONNECT;
+      // Run a blank command to get the user to connect first.
+      void vscode.commands.executeCommand('workbench.action.chat.open', {
+        query: '@MongoDB /query',
+      });
```

Comment:
Can we add a helper for this with more strongly defined types? I can see we're using it in a bunch of places and it's useful to extract the knowledge of the command in a single place.